code_samples/api/public_php_api/src/Command/ViewContentMetaDataCommand.php

```diff
@@ -99,10 +99,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         // Relations
         $versionInfo = $this->contentService->loadVersionInfo($contentInfo);
-        $relations = $this->contentService->loadRelations($versionInfo);
-        foreach ($relations as $relation) {
-            $name = $relation->destinationContentInfo->name;
-            $output->writeln('Relation to content ' . $name);
+        $relationCount = $this->contentService->countRelations($versionInfo);
```

Comment:
This could be replaced with 
```
        $relationListIterator = new BatchIterator(
            new RelationListIteratorAdapter(
                $this->contentService,
                $versionInfo
            )
        );
```
ref: https://github.com/ibexa/admin-ui/blob/main/src/lib/UI/Dataset/RelationsDataset.php#L52

Reply:
Really interesting. I'll explore that.

Reply:
It's absent from 4.6.13 https://github.com/ibexa/admin-ui/blob/v4.6.13/src/lib/UI/Dataset/RelationsDataset.php
Will it be in 4.6.14?

Reply:
Sadly no, not atm at least.

Reply:
For 5.0, I'll have #2548 ready